### ISSUE TITLE
Add option to set default video speed on invidious

### ISSUE THREAD

**Issue by sbstn87** [SimonBrazell/privacy-redirect#109]
Hi,

is it possible to add an option with a desired playback speed when redirecting to invidious?

Regards.

**Comment by SimonBrazell** [comment]
Yeah sure I think it's one of the Invidious parameters, I'll look at adding this when I next work on this project.
